### DISCUSSION TITLE
Disable scientific notation in response

### DISCUSSION THREAD

**Discussion by khaitranquang** [encode/django-rest-framework#7764] [Question & Answer] [answered]
Hi team,

I use DRF but I have an issue with float format in DRF Response. Here, my code:

```python
from rest_framework.response import Response

# Some code ....
result = {
  "origin_amount": 0.000000001,
  "origin_amount_text": "",
  "origin_currency": "USD",
  "fee": {
    "paypal_fee": 0.000000001,
    "vat": 0.000000001,
    "income_tax": 0.000000001,
    "whitehub_fee": 0.000000001
  },
  "total_amount": 0.000000001,
  "currency": "USD",
}
return Response(status=200, data=result)
```

I expetect it returns response as `result` dict. But it returns:
```
{"origin_amount": 1e-09, "origin_amount_text": "", "origin_currency": "USD", "fee": {"paypal_fee": 1e-09, "vat": 1e-09, "income_tax": 1e-09, "whitehub_fee": 1e-09}, "total_amount": 1e-09, "currency": "USD"}
```

So, how do i fix it?
My django and DRF version:
```
Django==2.0.5
djangorestframework==3.8.2
```

Tks

**Comment by ramwin** [comment, marked as answer]
This is not a problem with drf. It is the problem of json.  
You can see some questions [hear](https://stackoverflow.com/questions/1447287/format-floats-with-standard-json-module) or [hear](https://stackoverflow.com/questions/18936554/disable-scientific-notation-in-python-json-dumps-output).  
I suggest you use `DecimalField` in your code instead.
